static/js/core.js

```diff
@@ -1,226 +1,89 @@
-        } else {
-            Y.one('.nav-primary').insert('</a><a id="menu" class="nav-toggle">menu</a>', 'before');
-        }
+    if (navPrimary) {
+        // Add the navigation toggle
+        navToggle = document.createElement('a');
+        navToggle.classList.add('nav-toggle');
+        navToggle.id = 'menu';
+        navToggle.innerHTML = 'menu';
+        header.insertBefore(navToggle, navPrimary);
 
-        Y.all('.nav-toggle').on('click', function (e) {
+        // Adds a click listener to the navigation toggle
+        navToggle.addEventListener('click', function(e) {
             e.preventDefault();
-            if (Y.one('.header-search')) {
-                Y.one('.header-search').removeClass('active');
+            // Hide the search field
+            if (headerSearch) {
+                headerSearch.classList.remove('active');
             }
-
-            Y.all('.nav-toggle').toggleClass('active');
-            Y.all('header nav ul').toggleClass('active');
-            Y.all('.nav-primary').toggleClass('active');
+            // Toggle the navigations display state
+            navToggle.classList.toggle('active');
+            navPrimary.classList.toggle('active');
+            document.querySelector('#nav ul').classList.toggle('active');
         });
+    }
 
-        if (Y.one('.header-search')) {
-            Y.one('.search-toggle').on('click', function (e) {
-                e.preventDefault();
-
-                Y.all('.nav-toggle').removeClass('active');
-                Y.all('header nav ul').removeClass('active');
-                Y.all('.nav-primary').removeClass('active');
-
-                Y.one('.header-search').toggleClass('active');
-                Y.one('.header-search .form-text').focus();
-            });
-        }
-
-        if (Y.one('.breadcrumb li a')) {
-            Y.one('.breadcrumb > li a').insert('<span class="after"></span>');
-            Y.one('.breadcrumb li a .after').on('click', function (e) {
-                e.preventDefault();
-                Y.one('.nav-secondary').toggleClass('open');
-                core.setLocalStorage('onboard');
-            });
-        }
-
-    };
-    // REWRITE IN ES6
-    core.navOnboarding = function () {
-        var key = 'ubuntu',
-            nav_secondary = Y.one('.nav-secondary'),
-            ls;
-
-        if (nav_secondary && !nav_secondary.test(':empty')) {
-            if (Y.one('html').hasClass('localstorage')) {
-                if (localStorage.getItem(key) === null) {
-                    nav_secondary.addClass('open');
-                } else {
-                    ls = JSON.parse(localStorage.getItem(key));
-                    if (ls.navigation !== 'onboard') {
-                        nav_secondary.addClass('open');
-                    }
-                }
-            }
-        }
-    };
-    // NOT USING YUI
-    core.setLocalStorage = function ($value) {
```

Comment:
This stored the fact you had opened the second level navigation and closed it from then on.